Microsoft.Toolkit.Uwp.Notifications/DesktopNotificationManager/DesktopNotificationManagerCompat.cs

```diff
@@ -181,11 +181,11 @@ public static bool IsRunningAsUwp()
                     else
                     {
                         int length = 0;
-                        StringBuilder sb = new StringBuilder(0);
-                        int result = GetCurrentPackageFullName(ref length, sb);
+                        var sb = new StringBuilder(0);
+                        GetCurrentPackageFullName(ref length, sb);
 
                         sb = new StringBuilder(length);
-                        result = GetCurrentPackageFullName(ref length, sb);
+                        var result = GetCurrentPackageFullName(ref length, sb);
```

Comment:
change var back to int. Var makes it hard to understand what the type is in this situation

Reply:
As for the other comment, I've reverted the change and updated the variable name to have something more helpful than `result`